Source/SpatialGDK/Legacy/EntityRegistry.cpp

```diff
@@ -2,155 +2,58 @@
-                 *BlueprintName);
-          RegisterEntityClass(BlueprintName, BlueprintGeneratedClass);
-        }
-        else
-        {
-          UE_LOG(LogTemp, Warning,
-                 TEXT("Found asset in the EntityBlueprints folder which is not a blueprint: %s"),
-                 *(Asset->GetFullName()));
-        }
-      }
-    }
-    else
-    {
-      UE_LOG(LogTemp, Warning, TEXT("No assets found in EntityBlueprints folder."));
-    }
-  }
-}
-
-UClass** UEntityRegistry::GetRegisteredEntityClass(const FString& ClassName)
-{
-  auto ClassToSpawn = ClassMap.Find(ClassName);
-  if (ClassToSpawn == nullptr)
-  {
-    UE_LOG(LogEntityRegistry, Log, TEXT("No UClass is associated with ClassName('%s')."),
-           *ClassName);
-    return nullptr;
-  }
-
-  return ClassToSpawn;
-}
-
-void UEntityRegistry::AddToRegistry(const FEntityId& EntityId, AActor* Actor)
-{
-  EntityIdToActor.Add(FEntityId(EntityId), Actor);
-  ActorToEntityId.Add(Actor, EntityId);
+	EntityIdToActor.Add(EntityId, Actor);
+	ActorToEntityId.Add(Actor, EntityId);
 }
 
 void UEntityRegistry::RemoveFromRegistry(const AActor* Actor)
 {
-  FEntityId EntityId = GetEntityIdFromActor(Actor);
-  RemoveFromRegistryImpl(Actor, EntityId);
+	Worker_EntityId EntityId = GetEntityIdFromActor(Actor);
+	RemoveFromRegistryImpl(Actor, EntityId);
 }
 
-void UEntityRegistry::RemoveFromRegistry(const FEntityId& EntityId)
+void UEntityRegistry::RemoveFromRegistry(const Worker_EntityId& EntityId)
 {
-  AActor* Actor = GetActorFromEntityId(EntityId);
-  RemoveFromRegistryImpl(Actor, EntityId);
+	AActor* Actor = GetActorFromEntityId(EntityId);
+	RemoveFromRegistryImpl(Actor, EntityId);
 }
 
-void UEntityRegistry::RemoveFromRegistryImpl(const AActor* Actor, const FEntityId& EntityId)
+void UEntityRegistry::RemoveFromRegistryImpl(const AActor* Actor, const Worker_EntityId& EntityId)
 {
-  ActorToEntityId.Remove(Actor);
-  EntityComponentCache.Remove(Actor);
+	ActorToEntityId.Remove(Actor);
 
-  if (EntityId != -1)
-  {
-    EntityIdToActor.Remove(EntityId);
-  }
-  else
-  {
-    UE_LOG(LogEntityRegistry, Warning, TEXT("Couldn't remove Actor from registry: EntityId == -1"));
-  }
+	if (EntityId != -1)
+	{
+		EntityIdToActor.Remove(EntityId);
+	}
+	else
+	{
+		UE_LOG(LogEntityRegistry, Warning, TEXT("Couldn't remove Actor from registry: EntityId == -1"));
+	}
 }
 
-FEntityId UEntityRegistry::GetEntityIdFromActor(const AActor* Actor) const
+Worker_EntityId UEntityRegistry::GetEntityIdFromActor(const AActor* Actor) const
 {
-  if (ActorToEntityId.Contains(Actor))
-  {
-    return *ActorToEntityId.Find(Actor);
-  }
+	if (ActorToEntityId.Contains(Actor))
+	{
+		return *ActorToEntityId.Find(Actor);
+	}
 
-  return FEntityId();
+	return Worker_EntityId();
 }
 
-AActor* UEntityRegistry::GetActorFromEntityId(const FEntityId& EntityId) const
+AActor* UEntityRegistry::GetActorFromEntityId(const Worker_EntityId& EntityId) const
```

Comment:
`Worker_EntityId` is just a `std::int64_t`. Probably more performant to pass it by value.